File: 1/Solution.java
import java.util.Arrays;


public class Solution_1 {
  
  public int[] twoSum(int[] nums , int target){
    int length = nums.length;
    for (int i = 0 ; i < length ; i++){
      for (int j = i+1 ; j < length ; j++ ){
        if (nums[i] + nums[j] == target){
          return new int[]{i,j};  
        }
      }
    }
    return new int[0];
  }

  public int[] twoSum(int[] nums, int target) {
     Map<Integer,Integer> m = new HashMap<Integer,Integer>();
        for (int i = 0 ; i < nums.length ; i++){
            if (m.containsKey(target-nums[i])) return new int[]{m.get(target-nums[i]),i};
            m.put(nums[i] , i);
        }
        return new int[0];
    }
}

File: 1/Solution.cpp
#include <iostream> 
#include <array>
#include <vector>

using namespace std;

class Solution {
public:
    vector<int> twoSum(const vector<int>& nums , int target){
        for ( int i = 0 ;i < nums.size() ; i++)
            for ( int j = i+1; j< nums.size() ; j++){
                if (nums[i] + nums[j] == target){
                    return {i,j};
                };
            };
            return {};

    };
};


int main(){
    vector<int> nums = {1,3,4,5,6,8};
    int target;
    cin >> target;

    Solution solution;
    vector<int> result = solution.twoSum(nums,target);
    if (!result.empty()){
        cout<< "[" << result[0] <<","<<result[1]<<"]" << endl;
    } else {
        cout << "No found" << endl;
    }
    
    return 0;
}